util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -326,7 +326,7 @@ reg, mr, reg_idx)">\
 <%
     if compact_field_inst_name:
       reg_field_name = compact_field_inst_name
-      if len(fields) > 1:
+      if len(fields) > 1 or start_idx > 0:
```

Comment:
Do you think we can add a comment for this special handling?

Reply:
added a comment to explain the update. Thanks